### PR TITLE
removing the noValues for running devices endpoint

### PR DESCRIPTION
**Pull request by Baalmart**
**_WHAT DOES THIS PR DO?_**

- [x] removing the noValues for running devices endpoint
- [x] error handling and handling edge cases where Redis server is off

**_HOW DO I TEST OUT THIS PR?_**
```
cd src/device-registry
npm install
npm run dev-mac
```

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

- [x] list events
- [x] list running devices

### CODE REVIEW

File: src/device-registry/controllers/create-event.js [hunk 1]
```diff
@@ -642,12 +642,22 @@ const createEvent = {
       logObject("the result for listing events", result);
       if (result.success === true) {
         const status = result.status ? result.status : httpStatus.OK;
+        const devices = result.data[0].data || [];
+        const meta = result.data[0].meta;
+        if (devices && Array.isArray(devices)) {
```
Comment (Baalmart) [inline src/device-registry/controllers/create-event.js:647]: remove aqi_color fields for the running devices endpoint

File: src/device-registry/utils/create-event.js [hunk 2]
```diff
@@ -493,12 +460,28 @@ const createEvent = {
...
       const filter = generateFilter.events(request);
-      const cacheResult = await createEvent.getCache(request);
-      logObject("Cache result", cacheResult);
 
-      if (cacheResult.success === true) {
-        logText(cacheResult.message);
-        return cacheResult.data;
+      try {
+        const cacheResult = await Promise.race([
+          createEvent.getCache(request),
```
Comment (Baalmart) [inline src/device-registry/utils/create-event.js:466]: properly handle cases where the Redis server is off, just to ensure that the entire function does not hang. @BenjaminSsempala

File: src/device-registry/utils/create-event.js [hunk 3]
```diff
@@ -530,7 +513,26 @@ const createEvent = {
         data[0].data = !isEmpty(missingDataMessage) ? [] : data[0].data;
 
         logText("Setting cache...");
-        await createEvent.setCache(data, request);
+
+        try {
+          const resultOfCacheOperation = await Promise.race([
+            createEvent.setCache(data, request),
```
Comment (Baalmart) [inline src/device-registry/utils/create-event.js:519]: properly handle cases where the Redis server is off, just to ensure that the entire function does not hang. @BenjaminSsempala
